packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -277,6 +280,62 @@ const gateway = new appmesh.VirtualGateway(this, 'gateway', {
 });
 ```
 
+## Adding mutual TLS authentication
+
+To enable mutual TL authentication, add `certificate` property to TLS Client Policy and/or `validation` property to TLS Listener.
+
+A certificate from AWS Certificate Manager is **not** supported for mutual TLS.
+
+```typescript
+import * as certificatemanager from '@aws-cdk/aws-certificatemanager';
+
+// Validate a file client certificates to enable mutual TLS authentication when a client provides a certificate.
+const cert = new certificatemanager.Certificate(this, 'cert', {...});
+
+const node1 = new appmesh.VirtualNode(stack, 'node1', {
+  mesh,
+  serviceDiscovery: appmesh.ServiceDiscovery.dns('node'),
+  listeners: [appmesh.VirtualNodeListener.grpc({
+    port: 80,
+    tls: {
+      mode: appmesh.TlsMode.STRICT,
+      certificate: appmesh.TlsCertificate.acm({
+        certificate: cert,
+      }),
+      validation: {
+        trust: appmesh.TlsValidationTrust.file({
+          certificateChain: 'path-to-certificate',
+        }),
+      },
+    },
+  })],
+});
+
+// Provide a SDS client certificate when a server requests it and enable mutual TLS authentication.
+const certificateAuthorityArn = 'arn:aws:acm-pca:us-east-1:123456789012:certificate-authority/12345678-1234-1234-1234-123456789012';
+
+const node2 = new appmesh.VirtualNode(stack, 'node2', {
+  mesh,
+  serviceDiscovery: appmesh.ServiceDiscovery.dns('node'),
```

Comment:
not important since this is an example but different Virtual Nodes should have different DNS names

Reply:
Thanks for pointing this out! It is helpful to know these specific details.